widgets/common/src/filters/RuleOperatorDropdown.tsx

```diff
@@ -0,0 +1,20 @@
+import type { FunctionComponent } from 'react';
+
+const operators = [
```

Comment:
Optional:

How about adding a type annotation for the type of the values in the array, to make sure all array members are of the same shape? This way we are sure there are no typos in the objects :slightly_smiling_face: 

```suggestion
const operators: OperatorDescription[] = [
```

Reply:
The structure is not final. It can differ between attributes, so I think I will fix it once having proper structure.

Reply:
Sounds good :+1: